_posts/2020-01-22-biosemantics-versus-biopragmatics.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: Ben's first blog post
+date: 2020-09-01 00:00:00 -0800
+author: Ben Gyori
+---
+This is Ben's first blog post. Name the file with YYYY-MM-DD then some stuff that comes after
+that will be in the URL. The title and date in the frontmatter will be used for actual
+display and sorting on the blog.
+
+> TODO write a real blog post
```

Comment:
TODO write a real blog post! rename this file and change the date in the frontmatter too.